dockerfile/release.dockerfile

```diff
@@ -5,7 +5,7 @@ COPY ./requirements.txt requirements.txt
 
 RUN set -e -x \
     && apk --update --no-cache add \
-        bash gcc \
+        bash gcc curl \
```

Comment:
+100

Reply:
yeah - we may want to have development images vs. production images, e.g. dev images have additional tools (like vi) installed, production images have only the bare minimum needed to run.